canonical/ubuntu.com
# WD-5114 - Update the legal addresses across the site

## Done

- Update the legal addresses across the site

## QA

- Check that the addresses are correctly now pointing to Canonical Group Limited. Our address is 5 New street Square, London, EC4A 3TW, United Kingdom
- And 4th floor, 201 Borough High St. London SE1 1JA, United Kingdom for correspondence 
- Check the odd copy doc matches

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-5114
